users/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from .views import SignIn
+
+urlpatterns = [
+    path("",SignIn.as_view(),)
```

Comment:
주소를 조금 더 명확하게 적어주세요!
```suggestion
    path("/kakao/signin",SignInView.as_view(),)
```